src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    // !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs !!!
```

Comment:
If we want the file to be exactly the same, can't we somehow link it? For example, with [Paket](https://fsprojects.github.io/Paket/github-dependencies.html) we can reference a file directly.

Reply:
we don't need the identical, the meaning is that if we touch this one or the other side we need to keep in sync what we're using(I'll add to vstest repo as soon we merge this one).